hv-rhel7.x/hv/netvsc.c

```diff
@@ -895,7 +895,11 @@ int netvsc_send(struct net_device *ndev,
 	/* Keep aggregating only if stack says more data is coming
 	 * and not doing mixed modes send and not flow blocked
 	 */
+	#if (RHEL_RELEASE_CODE > RHEL_RELEASE_VERSION(7,1))
 	xmit_more = skb->xmit_more &&
+	#else
+	xmit_more = skb &&
```

Comment:
I'm not sure about the "<=7.1" case.    @haiyangz haiyang may comment on this.

Currently we have the below in netvsc_start_xmit():

        /* TODO: This will likely evaluate to false, since RH7 and
         * below kernels will set next pointer to NULL before calling
         * into here. Should find another way to set this flag.
         */
        packet->xmit_more = (skb->next != NULL);